tools/goctl/cmd/root.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/spf13/cobra"
+	"github.com/zeromicro/go-zero/tools/goctl/api"
+	"github.com/zeromicro/go-zero/tools/goctl/bug"
+	"github.com/zeromicro/go-zero/tools/goctl/docker"
+	"github.com/zeromicro/go-zero/tools/goctl/env"
+	"github.com/zeromicro/go-zero/tools/goctl/internal/version"
+	"github.com/zeromicro/go-zero/tools/goctl/kube"
+	"github.com/zeromicro/go-zero/tools/goctl/migrate"
+	"github.com/zeromicro/go-zero/tools/goctl/model"
+	"github.com/zeromicro/go-zero/tools/goctl/rpc"
+	"github.com/zeromicro/go-zero/tools/goctl/tpl"
+	"github.com/zeromicro/go-zero/tools/goctl/upgrade"
+)
+
+const codeFailure = 1
+
+var rootCmd = &cobra.Command{
+	Use:   "goctl",
+	Short: "A cli tool to generate go-zero code",
+	Long:  "A cli tool to generate api, zrpc, model code",
+}
+
+// Execute executes the given command
+func Execute() {
+	os.Args = supportGoStdFlag(os.Args)
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(aurora.Red(err.Error()))
+		os.Exit(codeFailure)
+	}
+}
+
+func supportGoStdFlag(args []string) []string {
+	copyArgs := append([]string(nil), args...)
+	parentCmd, _, err := rootCmd.Traverse(args[:0])
+	if err != nil {
+		return copyArgs
+	}
+
+	for idx, arg := range copyArgs {
+		if idx == 0 {
+			continue
+		}
```

Comment:
```suggestion
	for idx, arg := range copyArgs[0:] {
```
Maybe it is more simple?